src/network/graphParser.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash';
+
+class GraphParser {
+  static getParser(requestName) {
+    const PARSER_MAPPINGS = {
+      allTopics: this.parseTopic,
+      allOracles: this.parseOracle,
+      syncInfo: this.parseSyncInfo,
+    };
+    return PARSER_MAPPINGS[requestName];
+  }
+
+  static parseTopic(data) {
+    return data.map((entry) => ({
+      address: entry.address,
+      creatorAddress: entry.creatorAddress,
+      name: entry.name,
+      options: entry.options,
+      bettingEndBlock: entry.blockNum,
```

Comment:
is this correct?

Reply:
nope, removed. doesn't have that field anymore.